### PR TITLE
Add metadata

### PR DESCRIPTION
**Pull request by PieterJP**
Jira AT-1046 and AT-1151:
SpectrometerHDF5OutputFile.h and cpp:

1. Add noise diode temperature vs frequency tables from files for each band and for 5 and 20 kelvin
2. Add write csv to HDF

Add noise diode noise temperature vs frequency tables for band 1 and band2, 5 K and 20 K, LCP and RCP from files in NoiseDiode directory.

HDF5FileWriterh and cpp:

1. Add antenna beamwidth 
2. Add observation Information

KATCPServer.h and cpp:

1. Add antenna beamwidth 
2. Add observation Information
3. StationControllerKatcpClient.h and cpp:
4. Add antenna beamwidth 
5. Add observation Information

SpectrometerHDF5OutputFile.h and cpp:

1. Remove observation information from a file.
2. Add attributes for:
3. antenna name
4. antenna diameter
5. half power beamwidth
6. antenna longitude
7. antenna latitude
8. antenna altitude
9. observer
10. Conversion from DMS to degrees

### CODE REVIEW

**Comment by james-smith-za** [comment]
@PieterJP I am missing the context for this one?

**Review by rohanschwartz — APPROVED** [review]
LGTM, just remove commented out code, and make sure the SpectrometerDataStream submodule is coorrect.

File: StationControllerKATCPClient.cpp [hunk 1]
```diff
@@ -549,6 +551,21 @@ void cStationControllerKATCPClient::processKATCPMessage(const vector<string> &vs
 #endif
         return;
     }
+
+// PJP
+    if( !vstrTokens[3].compare("SCS.HPBW") )
+    {
+        cout << "±±§§§§ :" << vstrTokens[3] << " " << vstrTokens[5] << "\n";
```
[the remark this answers is not in the document]
Comment (PieterJP) [reply StationControllerKATCPClient.cpp:558]: No, remove

**Review by rishadebrahim — APPROVED** [review]
LGTM

**Pull request merged**